lib/v2/mrdx/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exportss = {
+    'mrdx.cn': {
+        _name: '新华每日电讯',
+        '.': [
+            {
+                title: '今日',
+                docs: 'https://docs.rsshub.app/traditional-media.html#xin-hua-mei-ri-dian-xun',
+                source: '',
```

Comment:
```suggestion
                source: ['*'],
```
You can add a wildcard match